src/Interpreters/TableJoin.h

```diff
@@ -153,6 +154,8 @@ class TableJoin
     ASTs key_asts_right;
 
     Clauses clauses;
+    /// Originally used for inequal join. If there is no any inequal join condition, it will be nullptr.
+    std::shared_ptr<ExpressionActions> full_join_expression = nullptr;
```

Comment:
`full` in join context may confuse, consider renaming it. Can't we also use `mixed_condition` here?

Reply:
This is missed in the last rename